rpc/api.go

```diff
@@ -126,6 +127,14 @@ func (sc *Client) ClassHashAt(ctx context.Context, address string) (string, erro
 	return *result, nil
 }
 
+// GetStorageAt gets the value of the storage at the given address and key.
+func (sc *Client) GetStorageAt(ctx context.Context, contractAddress, key, blockHashOrTag string) (string, error) {
```

Comment:
https://go.dev/doc/effective_go#Getters
```suggestion
func (sc *Client) StorageAt(ctx context.Context, contractAddress, key, blockHashOrTag string) (string, error) {
```